jar-infer/src/test/resources/com/uber/nullaway/jarinfer/testdata/Toys.java

```diff
@@ -0,0 +1,37 @@
+package toys;
```

Comment:
Is this file still used anywhere? If so, feel free to keep it, but if not, please remove.